File: Teste/app/src/main/java/com/example/rodolfo/teste/MainActivity.java
package com.example.rodolfo.teste;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void soma(View view){
        EditText n1 = (EditText) findViewById(R.id.n1);
        int num1 =  Integer.parseInt(n1.getText().toString());
        EditText n2 = (EditText) findViewById(R.id.n2);
        int num2 =  Integer.parseInt(n2.getText().toString());

        int soma =  num1 + num2;

        Toast.makeText(this, "Resultado: "+soma, Toast.LENGTH_LONG).show();
    }
}

File: Kotlin1/app/src/main/kotlin/MainActivity.kt
package com.example.rodolfo.kotlin1

import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.app.FragmentManager
import android.support.v7.widget.Toolbar
import com.example.rodolfo.kotlin1.FlowersFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val mToolbar = findViewById(R.id.my_toolbar) as Toolbar
        setSupportActionBar(mToolbar)
        mToolbar.setTitle("Teste Kotlin")

        if(savedInstanceState == null){
            changeFragment(FlowersFragment())
        }

    }

    fun changeFragment(f: Fragment, cleanStack: Boolean = false){
        val ft =  supportFragmentManager.beginTransaction()
        ft.setCustomAnimations(
                R.anim.abc_fade_in, R.anim.abc_fade_out, R.anim.abc_popup_enter, R.anim.abc_popup_exit);
        ft.replace(R.id.activity_base_content, f)
        ft.addToBackStack(null);
        ft.commit()
    }
}
